src/main/scala/akka/contrib/datareplication/ORMultiMap.scala

```diff
@@ -7,21 +7,21 @@ import akka.cluster.{ UniqueAddress, Cluster }
 
 object ORMultiMap {
 
+  val _empty: ORMultiMap[Any] = new ORMultiMap(ORMap.empty)
   /**
    * Provides an empty multimap.
    */
-  val empty: ORMultiMap =
-    new ORMultiMap(ORMap.empty)
-  def apply(): ORMultiMap = empty
+  def empty[A]: ORMultiMap[A] = _empty.asInstanceOf[ORMultiMap[A]]
+  def apply(): ORMultiMap[Any] = _empty
 
   /**
    * Java API
    */
-  def create(): ORMultiMap = empty
+  def create[A](): ORMultiMap[A] = empty[A]
 
   def unapply(value: Any): Option[Map[String, Set[Any]]] = value match {
```

Comment:
There can't be a way around it (the type argument), because the compiler can't take the type out of thin air (or pull it out of its ass, like Mark would say). I still prefer the typed alternative over the un(i)typed (i.e. `Any`) one.